lib/lightning_web/channels/run_channel.ex

```diff
@@ -94,7 +95,12 @@ defmodule LightningWeb.RunChannel do
          {:ok, scrubber} <- update_scrubber(scrubber, samples, basic_auth) do
       socket
       |> assign(scrubber: scrubber)
-      |> reply_with({:ok, remove_empty_values(credential.body)})
+      |> reply_with(
+        {:ok,
+         credential
+         |> assemble_body()
+         |> remove_empty_values()}
+      )
```

Comment:
I think we need to be really careful here:

1. I see we're changing the body we send to the client, that makes sense - but what about the scrubber, does the scrubber still work exactly the same way since it've been given a different structure?
2. I don't think we should manipulating the response body for the worker in this module, can we use or create a module that does rendering for these different structs?